cli/replicaset/cmd/status.go

```diff
@@ -73,9 +75,9 @@ func statusReplicasets(replicasets replicaset.Replicasets) error {
 	return nil
 }
 
-// fillAliases fills missed aliases with UUID. The case: Tarantool 1.10 without
+// FillAliases fills missed aliases with UUID. The case: Tarantool 1.10 without
 // an orchestrator.
-func fillAliases(replicasets replicaset.Replicasets) replicaset.Replicasets {
+func FillAliases(replicasets replicaset.Replicasets) replicaset.Replicasets {
```

Comment:
Why do we need to make the call public? The same for all new methods/structures.

Reply:
This is now a private method.